static/templates/rt_literal_property_attrs_doc.json

```diff
@@ -100,6 +100,50 @@
       }
     ]
   },
+  {
+    "@id": "_:b6",
+    "@type": ["http://sinopia.io/vocabulary/PropertyTemplate"],
+    "http://www.w3.org/2000/01/rdf-schema#label": [
+      {
+        "@value": "User ID Default Literal Data Type"
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasRemark": [
+      {
+        "@value": "Data type specifying a user ID default literal."
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasPropertyUri": [
+      {
+        "@id": "http://sinopia.io/vocabulary/userIdDefaultDataType"
```

Comment:
@justinlittman Is this an acceptable value? I made this up. Not sure if this should be the same or different from the URI in `static/userIdDefaultDataType.json`.